ale_linters/go/govet.vim

```diff
@@ -0,0 +1,46 @@
+" Author: neersighted <bjorn@neersighted.com>
+" Description: go vet for Go files
+
+if exists('g:loaded_ale_linters_go_govet')
+    finish
+endif
+
+let g:loaded_ale_linters_go_govet = 1
+
+function! ale_linters#go#govet#Handle(buffer, lines)
+    " Matches patterns line the following:
+    "
+    " file.go:27: missing argument for Printf("%s"): format reads arg 2, have only 1 args
+    let l:pattern = '^.*:\(\d\+\): \(.\+\)$'
```

Comment:
If you make the regex optionally look for a column number, you could apply the same handler for this linter too.